### DISCUSSION TITLE
Hacked up Intel NUC 11 Extreme control plugin

### DISCUSSION THREAD

**Discussion by crazynucfan** [Rem0o/FanControl.Releases#1598] [Show and tell]
Works for me on a NUC 11 Extreme, posting here in case someone else finds it useful or wants to adapt it for their NUC PCs

https://github.com/crazynucfan/FanControl.IntelNUCSoftwareStudioPlugin

**Comment by ricthot** [comment]
This is EXACLTY what I needed, thanks so much, works perfectly with my NUC 11X !

@Rem0o , this should be added to the list of plugins.
